package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "mercurius",
+  "version": "1.0.0",
+  "description": "Community Newsletter Initiative",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/osBins/Mercurius.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
We are using GPL 3.0 on our repository so I think we should use the same here.

Same applies for the licensing of the hugo theme.